opencti-platform/opencti-front/src/private/components/profile/Notifications.tsx

```diff
@@ -470,7 +459,13 @@ const Notifications: FunctionComponent = () => {
               {t_i18n('Delete')}
             </Button>
           </DialogActions>
-        </Dialog>)}
+        </Dialog>
+      )}
+      <DigestNotificationDrawer
+        notification={notificationDigestToOpen}
+        open={openNotifDrawer}
+        onClose={() => { setOpenNotifDrawer(false); }}
```

Comment:
_notificationDigestToOpen_ and _openNotifDrawer_ could be one variable (openNotifDrawer = !!notificationDigestToOpen)

```suggestion
{!!notificationDigestToOpen & (
      <DigestNotificationDrawer
        notification={notificationDigestToOpen}
        open={!!notificationDigestToOpen}
        onClose={() => { setNotificationDigestToOpen(null); }}
```